### PR TITLE
Translation Versions page

### PR DESCRIPTION
**Pull request by minimalsm**
## Description
Adds a page that explains what is in each translation version

### CODE REVIEW

**Comment by minimalsm** [comment]
@lukassim can you have a look at this and let me know your thoughts 🙏 

https://deploy-preview-3797--ethereumorg.netlify.app/en/contributing/translation-program/translation-versions/

File: src/content/contributing/translation-program/translation-versions/index.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+sidebar: true
+---
+
+# Translation Versions {#translation-versions}
+
+Below we have provided a breakdown of how each translation version changes [ethereum.org](/) in their localized language.
+
+## Version 2.0 {#version-20}
+
+Language V2.0 translates high-impact, highly trafficed pages on ethereum.org and the navigation components.
```
Comment (wackerow) [inline src/content/contributing/translation-program/translation-versions/index.md:14]:
```suggestion
Language V2.0 translates high-impact, highly trafficked pages on ethereum.org and the navigation components.
```

File: src/content/contributing/translation-program/translation-versions/index.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+### Primary Pages {#primary}
+
+- [Ethereum.org homepage](/)
+- [What is Ethereum](/what-is-ethereum/)
+- [What is Ether](/eth/)
```
Comment (wackerow) [inline src/content/contributing/translation-program/translation-versions/index.md:20]:
```suggestion
- [What is ether](/eth/)
```

File: src/content/contributing/translation-program/translation-versions/index.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+### Navigation {#navigation}
+
+- Main navbar
+- Footer navbar
+- Developer docs sidebar
+
+## Version 2.1 {#version-21}
+
+Language V2.1 translates the majority of the developer docs and several less trafficed but important pages.
```
Comment (wackerow) [inline src/content/contributing/translation-program/translation-versions/index.md:40]:
```suggestion
Language V2.1 translates the majority of the developer docs and several less trafficked but important pages.
```

File: src/content/contributing/translation-program/translation-versions/index.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+- [Token standards](/developers/docs/standards/tokens/)
+- [ERC-20](/developers/docs/standards/tokens/erc-20/)
+- [ERC-721](/developers/docs/standards/tokens/erc-721/)
+- [Orcles](/developers/docs/oracles/)
+- [Scaling](/developers/docs/scaling/)
+- [Layer 2 Rollups](/developers/docs/scaling/layer-2-rollups/)
+- [State channels](/developers/docs/scaling/state-channels)
+- [Sidechains](/developers/docs/scaling/sidechains)
+- [Plasma](/developers/docs/scaling/plasma)
+- [Validium](/developers/docs/scaling/validium/)
```
Comment (wackerow) [inline src/content/contributing/translation-program/translation-versions/index.md:106]: I'm fine with it either way, but any thoughts on simplifying this to just everything in "/developers/docs" is v2.1?

File: src/content/contributing/translation-program/translation-versions/index.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+- [How to turn your Raspberry Pi 4 into a node just by flashing the MicroSD card](/developers/tutorials/run-node-raspberry-pi/)
+- [Calling a smart contract from JavaScript](/developers/tutorials/calling-a-smart-contract-from-javascript/)
+- [Transfers and approval of ERC-20 tokens from s solidity smart contract](/developers/tutorials/transfers-and-approval-of-erc-20-tokens-from-a-solidity-smart-contract/)
+- [Interact wit hother contracts from Solidity](/developers/tutorials/interact-with-other-contracts-from-solidity/)
+- [Understand the ERC-20 token smart contract](/developers/tutorials/understand-the-erc-20-token-smart-contract/)
+- [Deploying your first smart contract](/developers/tutorials/deploying-your-first-smart-contract/)
+- [Logging data from smart contracts with events](/developers/tutorials/logging-events-smart-contracts/)
+- [How to implement an ERC-721 market](/developers/tutorials/how-to-implement-an-erc721-market/)
+- [How to use Manticore to find bugs in smart contracts](/developers/tutorials/how-to-use-manticore-to-find-smart-contract-bugs/)
```
Comment (wackerow) [inline src/content/contributing/translation-program/translation-versions/index.md:131]: Similar to above... should we consider just stating everything in "/developers/tutorials" is v2.2?

I suppose part of this depends on where newly listed tutorials and docs are going to be slotted for new content versions or not. If not, seems more dynamic and future proof to not explicitly list every page in these large folders.

**Review by wackerow — CHANGES_REQUESTED** [review]
Couple small adjustments suggested and a question about listing all the pages in docs and tutorials. 👍🏻 lookin good though

File: src/content/contributing/translation-program/translation-versions/index.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+
+# Translation Versions {#translation-versions}
+
+Below we have provided a breakdown of how each translation version changes [ethereum.org](/) in their localized language.
+
+## Version 2.0 {#version-20}
+
+Language V2.0 translates high-impact, highly trafficked pages on ethereum.org and the navigation components.
+
+### Primary Pages {#primary}
```
Comment (samajammin) [inline src/content/contributing/translation-program/translation-versions/index.md:16]: There's a bunch missing: https://crowdin.com/project/ethereum-org/content/files

**Review by samajammin — APPROVED** [review]
Great idea & nice work on this @minimalsm!

Made some updates:
- Updated page from "Translation versions" to "Content versions"
- Added a brief overview section
- Added link from Translation Program page
- Added missing pages from v2.0
- Simplified lists to list "all docs pages" & "all tutorial pages"

I think this is ready to merge but feel free to make any edits you'd like first.

**Review by corwintines — APPROVED** [review]

**Review by wackerow — APPROVED** [review]

**Comment by minimalsm** [comment]
Bringing this in for the deploy. Will make necessary changes in a patch deploy 👍

**Pull request merged**